.github/workflows/clang-tidy.yml

```diff
@@ -42,12 +42,12 @@ jobs:
         with:
           conan_dir: ${{ env.CONAN_USER_HOME }}/.conan
           ccache_dir: ${{ env.CCACHE_DIR }}
-          conan_profile: ${{ steps.conan.outputs.conan_profile }}
+          conan_profile: clang
```

Comment:
But that's why it was originally created. Now you have to specify the same profile everywhere and make sure it is the same.

Reply:
Thank you for suggestion, used env variable and removed explicit conan_profile value duplication! 👍